sashimi-webapp/src/database/sql-related/sqlCommands.js

```diff
@@ -0,0 +1,261 @@
+import StringManipulator from '../stringManipulation';
+
+import exceptions from '../exceptions';
+
+import constants from '../constants';
+
+import DateTime from '../generated-data/dateTime';
+
+const alasql = require('alasql');
```

Comment:
And about the import statement, I guess I will update because previously when I was coding the eslint was giving red flag that there should be a space and when I save it auto generated new lines.